configs/default/FOSS-DRONIUSF7.config

```diff
@@ -0,0 +1,79 @@
+# version
+# Betaflight / DRONIUSF7 (FOSS) 4.1.5 Mar  9 2020 / 16:02:05 (7a5ffad37) MSP API: 1.42
```

Comment:
This is not a valid Unified Target - installing this will not work.

Can you please test the contents of your pull requests before you submit them.